.github/workflows/example_test.yaml

```diff
@@ -2,11 +2,17 @@ name: KubeagiTests
 
 on:
   push:
-    branches: 
+    branches:
       - 'main'
   pull_request:
     branches: [main]
   workflow_dispatch:
+    inputs:
```

Comment:
Can you provide more details on this change ?

Reply:
See “Special notes for your reviewer” on pull request comment.

Please feel free to tell me what I didn't explain clearly～